### PR TITLE
HPCC-12532 Superfile delete/remove confusion

### PR DESCRIPTION
**Pull request by GordonSmith**
Add clearer button names.
Disable remove if nothing selected.

Fixes HPCC-12532

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/templates/SFDetailsWidget.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
                     <div id="${id}Refresh" data-dojo-attach-event="onClick:_onRefresh" data-dojo-props="iconClass:'iconRefresh'" data-dojo-type="dijit.form.Button">${i18n.Refresh}</div>
                     <span data-dojo-type="dijit.ToolbarSeparator"></span>
                     <div id="${id}Save" data-dojo-attach-event="onClick:_onSave" data-dojo-type="dijit.form.Button">${i18n.Save}</div>
-                    <div id="${id}Delete" data-dojo-attach-event="onClick:_onDelete" data-dojo-type="dijit.form.Button">${i18n.Delete}</div>
+                    <div id="${id}Delete" data-dojo-attach-event="onClick:_onDelete" data-dojo-type="dijit.form.Button">${i18n.DeleteSuperfile2}</div>
```
Comment (miguelvazq) [inline esp/src/eclwatch/templates/SFDetailsWidget.html]: Should we not be asking the question here? For example --> ${i18n.DeleteSuperfile}?

**Comment by miguelvazq** [comment]
Just the minor comment above. All else looks good.

File: esp/src/eclwatch/templates/SFDetailsWidget.html (hunk 1 above)
Comment (GordonSmith) [inline esp/src/eclwatch/templates/SFDetailsWidget.html]: When the user clicks this button they will be presented with the confirmation dialog which does ask the question - but the text here is just for the button and should not have a "?".

**Pull request merged**